lib/pure/times.nim

```diff
@@ -315,6 +315,10 @@ type
                      ##
                      ## To create a new `Duration`, use `initDuration proc
                      ## <#initDuration,int64,int64,int64,int64,int64,int64,int64,int64>`_.
+                     ## To access private attributes, use
+                     ## `inSeconds proc<#inSeconds,Duration>`_,
+                     ## `inNanoseconds proc<#inNanoseconds,Duration>`_ and etc.
+                     ## 
```

Comment:
```suggestion
                     ## To create a new `Duration`, use `initDuration
                     ## <#initDuration,int64,int64,int64,int64,int64,int64,int64,int64>`_.
                     ## Instead of trying to access the private attributes, use
                     ## `inSeconds <#inSeconds,Duration>`_ for converting to seconds and
                     ## `inNanoseconds <#inNanoseconds,Duration>`_ for converting to nanoseconds.
```
If you look at the implementation, you see that `inSeconds`, `inNanoseconds`, etc. don't just access the private attributes, they also do a conversion.

Also, nit: you can move the doc comment, so that it isn't so far on the right:
```nim
  Duration* = object
    ## ...
    seconds: int64
    nanosecond: NanosecondRange

Reply:
thanks